install-hooks.sh

```
@@ -0,0 +1,458 @@
+
+        write_status "Downloading: $url" "INFO"
+
+        if curl -fsSL "$url" -o "$dest" 2>/dev/null; then
+            chmod +x "$dest"
+            write_status "✓ Downloaded: $(basename "$dest")" "SUCCESS"
+        else
+            write_status "⚠ Failed to download $url" "ERROR"
+            write_status "Using local package files instead..." "INFO"
+
+            # Fallback: Copy from npm package
+            local npm_hooks="$MCP_GLOBAL_PATH/hooks"
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)